sdk/packages/eternum/src/constants/contracts.ts

```diff
@@ -11,6 +11,7 @@ export const DEV_CONTRACTS = {
   TRAVEL_SYSTEMS: "0x70717be365c143d9f4ae207e420d0c3525a7c79197a20e6e63e4dec0b1b26cd",
   TEST_REALM_SYSTEMS: "0xee850f8d8ded18763d23e35d3592f9549081124bee0c32ffb2ef355deb1b68",
   TEST_RESOURCE_SYSTEMS: "0x17505387265fa24a4846cb3b3207aa92c3e6f918f74883a43be959265d19e44",
+  CC_CONTRACT_ADDRESS:"0x1e26e39c8d6d9d03179426d6a95190001b72a75645da0a3cff264c936bd3f1",
```

Comment:
why are production and development the same?

Reply:
How to get Production Contract Address